app/views/layouts/application.html.erb

```diff
@@ -4,12 +4,6 @@
 
 <%= extends_layout :base, body_class: local_assigns.fetch(:body_class, 'site tablet:bg-primary-lighter') do %>
   <%= link_to t('shared.skip_link'), '#main-content', class: 'usa-skipnav' %>
-  <div class="usa-overlay"></div>
-  <% if content_for?(:mobile_nav) %>
-    <div class="usa-header">
-      <%= yield(:mobile_nav) %>
-    </div>
-  <% end %>
```

Comment:
Previously I might have assumed `usa-overlay` could be used for things other than mobile navigation (e.g. modals) and that's why it always needed to be present, but in reality it is very specifically for header/navigation.

See: https://github.com/search?q=repo%3Auswds%2Fuswds%20usa-overlay&type=code